swo_aws_extension/flows/steps/create_new_aws_environment.py

```
@@ -0,0 +1,111 @@
+
+    def __init__(self, config: Config):
+        self._config = config
+
+    @override
+    def pre_step(self, context: PurchaseContext) -> None:
+        phase = get_phase(context.order)
+        if phase != PhasesEnum.CREATE_NEW_AWS_ENVIRONMENT:
+            raise SkipStepError(
+                f"{context.order_id} - Next - Current phase is '{phase}', skipping as it"
+                f" is not '{PhasesEnum.CREATE_NEW_AWS_ENVIRONMENT}'"
+            )
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)